<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmPngTool</class>
 <widget class="QWidget" name="frmPngTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLineEdit" name="txtFile"/>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="btnOk">
     <property name="text">
      <string>执行转换</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="txtDir"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btnDir">
     <property name="text">
      <string>选择目录</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="btnFile">
     <property name="text">
      <string>选择文件</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QProgressBar" name="progress">
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QTextEdit" name="txtMain"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
